metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/publish/DefaultLogPublisher.java

```diff
@@ -106,83 +117,117 @@ protected void printThreadPoolMetrics(DefaultPublishModel model, StringBuilder s
     }
   }
 
+  protected void printEdgeLog(DefaultPublishModel model, StringBuilder sb) {
+    OperationPerfGroups edgePerf = model.getEdge().getOperationPerfGroups();
+    if (edgePerf == null) {
+      return;
+    }
+    sb.append(String.format(SIMPLE_HEADER, "edge"));
+    //clear old data
+    MeterStatisticsManager.statisticsOperationMap.clear();
```

Comment:
why need global variable?

Reply:
I just use it to load data temporarily as I need restructure the data

Reply:
I have removed the global variable